cdap-cli/src/main/java/co/cask/cdap/cli/command/UseNamespaceCommand.java

```diff
@@ -31,16 +32,21 @@
 public class UseNamespaceCommand extends AbstractAuthCommand {
 
   private final CLIConfig cliConfig;
+  private final NamespaceClient namespaceClient;
+
 
   @Inject
-  public UseNamespaceCommand(CLIConfig cliConfig) {
+  public UseNamespaceCommand(CLIConfig cliConfig, NamespaceClient namespaceClient) {
     super(cliConfig);
     this.cliConfig = cliConfig;
+    this.namespaceClient = namespaceClient;
   }
 
   @Override
   public void perform(Arguments arguments, PrintStream output) throws Exception {
     String namespace = arguments.get(ArgumentName.NAMESPACE_ID.toString());
+    // Check if namespace exists; throws exception if namespace doesn't exist.
+    namespaceClient.get(namespace);
```

Comment:
CLI should not show exception to user. Will it show proper error message saying "Failed to change to <namespace> as the <namepsace> doesn't exist"